packages/kujira-fin/src/execute.rs

```diff
@@ -22,13 +22,10 @@ pub enum ExecuteMsg {
         fee_maker: Option<Decimal256>,
     },
 
+    SetOracles(Option<(String, String)>),
```

Comment:
Shouldn't this be a part of `UpdateConfig` as `Option<Option<(String, String)>>`?

Problem is differentiating between `Some(None)` and `None` during the update. Maybe

```rust
#[cw_serde]
pub enum Oracles {
     None,
     Some(String, String)
}

// later
UpdateConfig {
    oracles: Option<Oracles>
}
```

Reply:
I would also add to instantiate message as `Option<Oracles>` -- Shouldn't break any existing interfaces (None if omitted).